app/views/posts/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Post Index Page</h1>
\ No newline at end of file
+<h1>Post Index Page</h1>
+<div>
+<h2>User</h2>
+  <img src="#{<%= @user.photo %>}" alt="User Img" />
+  <span><%= @user.name %></span>
+  <span><%= @user.posts_counter %></span>
+</div>
+<div>
+<h2>Posts</h2>
+<ol>
+<% @posts.each do |post| %>
+<li>
+<p><%= post.title%></p>
+<p><%= post.text%></p>
+<p>number on comments:<%= post.comment_counter || 0%></p>
+<p>number on likes:<%= post.likes_counter || 0%></p>
+</li>
```

Comment:
- [ ] Kindly use `post.recent_comments` to render 5 most recent comments as in this [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABYH576Q6LJTZRQ6P5CMNUC2YZUPHSA)

Reply:
Thanks.

Reply:
Added 👍🏻